ci/windows/inside_mingw_docker.sh

```diff
@@ -59,6 +59,7 @@ deploy_app(){
 	mkdir $DEST_FOLDER
 	cp $BUILD_FOLDER/Scopy.exe $DEST_FOLDER/
```

Comment:
rename target to scopy.exe (lowercase) or rename the other target to Scopy-cl.exe
On that note, would scopy-console.exe be a better name  ?
Scopy-Console ?